### ISSUE TITLE
请问最后的准确率在多少？

### ISSUE THREAD

**Issue by charlesfufu** [JNhua/Chinese_NER_with_Attention#1]
感觉准确率有点低哈

**Comment by aslicedbread** [comment]
> 感觉准确率有点低哈

我尝试打印了alpha_i的值，全是1（因为e_i = tf.reshape(e_i, [-1, s[1], 1])，alpha_i = tf.nn.softmax(e_i)，e_i的第三维度为1），我感觉这个attention可能写的有点问题。

**Comment by cyl250** [comment]
你们会不会多出一个类别啊，，，，训练着训练着

**Comment by maybefeicun** [comment]
楼主你这个代码的问题修改了没，我跑的时候不能准确的进行验证，感觉代码有点问题。您是在哪进行参考的可以透露下吗，感谢

**Comment by chengming1108** [comment]
@aslicedbread    我也在看这个att  我刚开始学这块的东西   您方便告诉我怎们打印alpha与e_i的值么  我想研究一下好在项目里用

**Comment by yexm** [comment]
楼主 想问一下   如何验证呢 你这里加入了词性
